pages/common/jdeps.md

```diff
@@ -0,0 +1,24 @@
+# jdeps
+
+> Java Class Dependency Analyzer.
+> More information: <https://docs.oracle.com/en/java/javase/17/docs/specs/man/jdeps.html>.
+
+- Analyze the dependencies of a `.jar` or `.class` file:
+
+`jdeps {{path/to/filename.class}}`
+
+- Print a summary of all dependencies:
+
+`jdeps {{path/to/filename.jar}} -summary`
+
+- Print all class-level dependencies:
+
+`jdeps {{filename.jar}} -verbose`
+
+- Output the results of the analysis in form of a DOT file:
+
+`jdeps {{path/to/filename.jar}} -dotoutput {{output/path}}`
```

Comment:
```suggestion
`jdeps {{path/to/filename.jar}} -dotoutput {{path/to/output}}`
```
I think this sounds better.

> Does this command require a file extension?

Reply:
It expects the path to a directory to place generated files into, so I was thinking of just changing it to `path/to/directory` which seems to be the standard on other pages.

Reply:
I made that change along with the other suggestions

Reply:
Thanks